### PR TITLE
Add consent functionality

### PR DESCRIPTION
**Pull request by niamh-d**
Added following two new methods to Processor class:
- revokeConsent
- verifyConsent 

 Added relevant unit tests

### CODE REVIEW

File: test/Processer.test.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    test('A user older than 18 can give consent; consentGiven field will be true and checkConsent method will return true', () => {
+      // default age for user = 21
+      processor.giveConsent(user)
+
+      expect(user.consentGiven).toBe(true)
+      expect(processor.checkConsent(user)).toBe(true)
+    })
+
+    test('revokeConsent method revokes consent; consentGiven field will be false and checkConsent method will return false', () => {
+      processor.giveConsent(user)
```
Comment (vladimir-tl) [inline test/Processer.test.ts:70]: general comment: test name too long, we don't have to describe all steps just main idea

File: src/Processor.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { User } from './User'
+
+export class Processor {
+  private readonly AGE_MAJORITY_YRS = 18
+
+  public giveConsent(userInput: User): void {
+    if (!this.verifyAge(userInput)) return
+
```
Comment (vladimir-tl) [inline src/Processor.ts:8]: pls delety empty lines here and other places

**Review by vladimir-tl — CHANGES_REQUESTED** [review]
general: Consider adding more descriptive commit messages for better traceability of changes.

File: test/Processer.test.ts (hunk 1 above)
Comment (niamh-d) [reply test/Processer.test.ts:70]: done

File: src/Processor.ts (hunk 2 above)
Comment (niamh-d) [reply src/Processor.ts:8]: done

**Pull request merged**